here-naksha-app-service/src/main/resources/static/openapi.yaml

```diff
@@ -247,6 +247,63 @@ paths:
           $ref: '#/components/responses/ErrorResponse504'
         '513':
           $ref: '#/components/responses/ErrorResponse513'
+  '/hub/spaces':
+    post:
+      tags:
+        - Manage Spaces
+      summary: Create new Space
+      description: Create new Space
+      operationId: postSpace
+      responses:
+        '201':
+          $ref: '#/components/responses/HandlerResponse'
+        '401':
+          $ref: '#/components/responses/ErrorResponse401'
+        '403':
+          $ref: '#/components/responses/ErrorResponse403'
+        '404':
+          $ref: '#/components/responses/ErrorResponse404'
+        '429':
+          $ref: '#/components/responses/ErrorResponse429'
+        '500':
+          $ref: '#/components/responses/ErrorResponse500'
+        '502':
+          $ref: '#/components/responses/ErrorResponse502'
+        '504':
+          $ref: '#/components/responses/ErrorResponse504'
+        '513':
+          $ref: '#/components/responses/ErrorResponse513'
+  '/hub/spaces/{spaceId}':
+    put:
+      tags:
+        - Manage Spaces
+      summary: Update a Space with given ID
+      description: Update already existing Space
+      operationId: putSpace
+      parameters:
+        - name: spaceId
+          in: path
+          description: ID of Space to update
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: '#/components/responses/HandlerResponse'
```

Comment:
the response should also be changed here

Reply:
nice catch, fixed :)